Assignment2/juanbm99-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,42 @@
+{
+	"@context": {
+			"vocab": "http://www.alumnosSemanticWeb.es/alumnos#",
+			"nombre_Completo": "vocab:nombre_Completo",
+			"localidad": "vocab:localidad",
+			"edad": "vocab:edad",
+			"almaMater": "vocab:almaMater",
+			"nombre_Padre": "vocab:nombre_Padre",
+			"nombre_Madre": "vocab:nombre_Madre",
+			"asignaturas": "vocab:asignaturas",
+			"nombre_asignatura": "asignaturas:nombre_asignatura",
+			"profesores": "asignaturas:profesores",
+			"descripcion": "asignaturas:descripcion"
```

Comment:
These subject attribute definitions work, but semantically they aren't ok. The best way to define them might be defining a new prefix for the asignaturas concept and then use it to define their attributes. e.g. "assignatura": "http://www.alumnosSemanticWeb.es/asignatura#",  "nombre_asignatura": "asignatura:nombre_asignatura", etc.

Reply:
You don't need to change them, but remember to provide good definitions for the future.